packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -192,8 +192,9 @@ export class VerticalStackedBarChartBase extends React.Component<
       .range([this._height, 0]);
     const yAxis = d3AxisLeft(yAxisScale)
       .tickSizeInner(-this._width)
-      .tickPadding(10)
-      .tickValues(domains);
+      .tickPadding(5)
+      .tickValues(domains)
+      .ticks(this._yAxisTickCount, 's');
```

Comment:
@Raghurk , Charts should format the values so that large values get converted to SI units, like K,M,G,T.